src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/unit/restapi/transformer/ReferralTransformersTest.kt

```diff
@@ -23,10 +25,6 @@ import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.domain.entity.u
 
 class ReferralTransformersTest {
 
-  companion object {
```

Comment:
Why has this gone?

Reply:
ah looks like it's removed in a couple of commits' time

Reply:
Ah, must've put that change in the wrong commit. Apologies. Thought I'd caught them all.